File: src/main/java/it/contrader/dto/FindAWordDTO.java
//created by Gabriella Brunetto

package it.contrader.dto;

import com.fasterxml.jackson.annotation.JsonIdentityInfo;
import com.fasterxml.jackson.annotation.ObjectIdGenerators;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;



@Data
@AllArgsConstructor
@NoArgsConstructor
@JsonIdentityInfo(generator=ObjectIdGenerators.PropertyGenerator.class, property="id")
public class FindAWordDTO {
		private static final String typeGame = "FindAWord";
	
		private Long id;
		
		private CategoryDTO category;
		
		private LevelDTO level;
		
		private String solution;
		
		private String definition;
		
		private String sentence;

		public static String getTypeGame() {
			return typeGame;
		}
}

File: src/dto/findaworddto.ts
import {CategoryDTO} from "./categorydto";
import {LevelDTO} from "./leveldto";


export class FindAWordDTO{

    static typeGame: string = "FindAWord";
	
    id: number;
    
    category: CategoryDTO;
    
    level: LevelDTO;
    
    solution: string ;
    
    definition: string;
    
    sentence: string ;

    categoryTitle: string;
    levelName: string;

}


